src/pages/settings/invoice-design/pages/general-settings/components/CompanyAddress.tsx

```diff
@@ -13,7 +13,7 @@ import { useTranslation } from 'react-i18next';
 import { SortableVariableList } from './SortableVariableList';
 import { useCustomField } from '$app/components/CustomField';
 
-export function CompanyAddress() {
+export default function CompanyAddress() {
```

Comment:
I see we started default exporting the functions. Why?

Reply:
@beganovich The reason for doing this is actually lazy loading. I found that we had already implemented lazy loading on the main `InvoiceDesign` component and `CustomDesigns` components. However, they were only linked components. Once I made `ClientDetails` and the rest of them linked tabs, I decided to follow this pattern and implement lazy loading on those as well. I hope that you will agree with following the already implemented pattern.